ios/CodePush/CodePushPackage.m

```diff
@@ -39,8 +39,22 @@ + (void)downloadAndReplaceCurrentBundle:(NSString *)remoteBundleUrl
     }
 }
 
++ (void) handleFailedDataIntegrityCheck:(void (^)(NSError *err))failCallback
+                                   error:(NSError **)error
+{
+    CPLog(@"The update contents failed the data integrity check.");
+    if (!error || !*error) {
```

Comment:
Not sure if `!*error` is needed here, I believe `if (!error)` should be enough.

Reply:
Also don't understand why do we need at all `err` variable, seems that it is not used anywhere at all

Reply:
> Also don't understand why do we need at all err variable, seems that it is not used anywhere at all

this is a part of callback function signature

Reply:
>Not sure if !*error is needed here, I believe if (!error) should be enough.

Okay, i will refactor it to simplify conditions